metadata/economy/enterprise_zones.md

```diff
@@ -0,0 +1,74 @@
+# Title
+
+Utah Enterprise Zones
+
+## ID
+
+2468fb97-fc82-4887-8aa0-60ec91f66b6a
+
+## Brief Summary
+
+Polygon dataset of enterprise zones in Utah as defined by the Utah Governor's Office of Economic Opportunity.
+
+## Summary
+
+The Utah Enterprise Zone Tax Credit program was created to promote economic development and business growth in Utah's rural areas. Under this program, qualifying businesses locating to or expanding in designated enterprise zones may claim state of Utah tax credits. This dataset contains polygon boundaries of current enterprise zones in Utah.
+
+## Description
+
+### What is the dataset?
+
+An enterprise zone is an area designated by the [Utah Governor's Office of Economic Opportunity](https://business.utah.gov/rural/enterprise-zone-tax-credits/) that provides income tax credits to eligible businesses within its boundaries. Zone designation is based on the need for development and other economic distress factors as determined by local elected officials. These zones were established in 1988 as part of the Utah Enterprise Zone Tax Credit Program to encourage economic growth and support businesses in Utah's rural areas.
+
+### What is the purpose of the dataset?
+
+This dataset has been made available to the public for general analytic and cartographic purposes.
+
+### What does the dataset represent?
+
+Each polygon in this dataset represents the geographic extent of an enterprise zone. Features in this dataset indicate the zone name, county, and expiration year, as well as the email and phone number of the zone point of contact.
+
+### How was the dataset created?
+
+UGRC created this dataset using legal descriptions verified through the Utah Governor's Office of Economic Opportunity. All updates and adjustments to these boundaries must first by certified by the state.
```

Comment:
This should probably say something like "This dataset was created with boundaries provided and verified by GOEO"